contracts/AcceleratingDistributor.sol

```diff
@@ -127,9 +127,9 @@ contract AcceleratingDistributor is ReentrancyGuard, Ownable, Multicall {
         // benefit to removing this additional flexibility. If set < 1e18, then user's rewards outstanding will
         // decrease over time. Incentives for stakers would look different if `maxMultiplier` were set < 1e18
         require(stakedToken != address(rewardToken), "Staked token is reward token");
-        require(maxMultiplier < 1e36, "maxMultiplier can not be set too large");
-        require(secondsToMaxMultiplier > 0, "secondsToMaxMultiplier must be greater than 0");
-        require(baseEmissionRate < 1e27, "baseEmissionRate can not be set too large");
+        require(maxMultiplier <= 10e18, "maxMultiplier too large"); // 1000% realistic upper bound.
```

Comment:
I think 1e36 is probably a bit ridiculous, but I think 10x multiplier is probably pretty low for a _contract-level_ restriction. I could imagine this contract being used in ways we can't foresee where a larger multiplier over a very long time may be desired.

I think 1,000,000x (i.e. 1e24) might be a decent middle ground. Thoughts?

Reply:
hmm ok i'm preetty unopinonated and this is many many orders of magnitude more reasonable than 1e36